static/js/containers/DashboardPage.js

```diff
@@ -613,11 +614,17 @@ class DashboardPage extends React.Component {
       this.shouldSkipFinancialAid() ||
       program.financial_aid_user_info.has_user_applied
     );
+    let pendingFinancialAid = (
```

Comment:
Can you add a test for this? We have a test that `pendingFinancialAid` is used by `CourseEnrollmentDialog` but there's no test for what situatlions this value is calculated